draft-ietf-mls-protocol.md

```diff
@@ -4226,6 +4236,21 @@ extensible.  As discussed in {{commit}}, Proposals with a non-default proposal
 type MUST NOT be included in a commit unless the proposal type is supported by
 all the members of the group that will process the Commit.
 
+## Credentials
+
+The types of credential that may be used in a group is restricted to what all
+members of the group support, as specified by the `capabilities` field of each
+LeafNode in the RatchetTree. An application can introduce new credential types
+by choosing an unallocated identifier from the registry in
+{{mls-credential-types}} and indicating support for the credential type in
+published LeafNodes, whether in Update proposals to existing groups or
+KeyPackages that are added to new groups. Once all members in a group indicate
+support for the credential type, members can start using LeafNodes with the new
+credential. Application may enforce that certain credential types always remain
+supported by adding a `required_capabilities` extension to the group's
+GroupContext, which would prevent any member from being added to the group that
+doesn't support them.
+
```

Comment:
```suggestion

In future extensions to MLS, it may be useful to allow a member to present more
than one credential.  For example, such credentials might present different
attributes attested by different authorities.  To be consistent with the general
principle stated at the beginning of this section, such an extension would need
to ensure that each member can authenticate some identity for each other member.
For each pair of members (Alice, Bob), Alice would need to present at least one
credential of a type that Bob supports.
```